app/src/main/kotlin/org/cru/godtools/ui/dashboard/lessons/LessonsLayout.kt

```diff
@@ -30,18 +30,23 @@ internal sealed interface DashboardLessonsEvent {
 @OptIn(ExperimentalFoundationApi::class)
 internal fun LessonsLayout(viewModel: LessonsViewModel = viewModel(), onEvent: (DashboardLessonsEvent) -> Unit = {}) {
     val lessons by viewModel.lessons.collectAsState()
+    val selectedLanguage by viewModel.selectedLanguage.collectAsState()
+
     LazyColumn(contentPadding = PaddingValues(16.dp)) {
         item("header", "header") { LessonsHeader() }
 
+        item("filter", "filter") { LessonFilters() }
```

Comment:
let's just include the composable call to `LessonFilters()` in the `item("header", "header")` item